index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
```

Comment:
Hi! Nice to meet you! I'm Estefanía and I'll be taking a look at your site's code to see how can we make it functional and shiny.👋😀.